File: tracker/src/test/java/testhelpers/TestHelper.java
package testhelpers;

import timber.log.Timber;

public class TestHelper {

    public static void sleep(long millis) {
        try {
            Thread.sleep(millis);
        } catch (InterruptedException e) {
            e.printStackTrace();
            Timber.e(e);
        }
    }
}

File: tracker/src/test/java/testhelpers/TestHelper.kt
package testhelpers

import timber.log.Timber

object TestHelper {
    @JvmStatic
    fun sleep(millis: Long) {
        try {
            Thread.sleep(millis)
        } catch (e: InterruptedException) {
            Timber.e(e)
        }
    }
}
